### DISCUSSION TITLE
`FluentNumberField` Validation Error on Submit

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [microsoft/fluentui-blazor#1849] [Q&A] [answered]
I am encountering the following when submitting a form with a `FluentNumberField` on it:

![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/f54adee0-5c68-450b-ae14-0db73ec8f121)

This is the code that I am using:

```
<FluentNumberField @bind-Value="Input.Price" Placeholder="Default Price for Products Created in this Topic"
                           Min=".5"
                           Max="100" Step=".25" style="width: 100%;" />
```
`Input.Price` is a `decimal`.

It appears that the number is resolving to `int` despite the field type being `decimal`.  I've also tried explicitly stating `FluentNumberField TValue="decimal"` as well, with the same results.

I am curious if there is a consideration here to resolve this.  Thank you for any assistance you can provide.

**Comment by vnbaaij** [comment, marked as answer]
Hi, this is a known issue in the underlying web component. It needs to be solved there first. 

Use the search to find related issues and  discussions

**Reply by Mike-E-angelo** [reply]
FWIW: https://github.com/microsoft/fluentui-blazor/issues/1175
